CleanDevelopment.sln

```diff
@@ -20,14 +20,14 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "docs", "docs", "{8715F244-D
 		README.md = README.md
 	EndProjectSection
 EndProject
-Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Component.Tests", "tests\Component.Tests\Component.Tests.xproj", "{69F5C049-59D2-448C-B009-8F1C48D537C5}"
-EndProject
-Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Components", "src\Components\Components.xproj", "{FDB68596-4408-419D-87A0-A6A170DC8144}"
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "EntityComponentSystem", "src\Components\EntityComponentSystem.xproj", "{FDB68596-4408-419D-87A0-A6A170DC8144}"
```

Comment:
Don't feel bad. As simple as this task might have looked at first glance, there's a tonne of things to change in a tonne of different places. You were never going to spot them all, and even after review we'll probably have missed something. ;)